catalog/src/main/java/org/killbill/billing/catalog/DefaultLimit.java

```diff
@@ -40,9 +41,19 @@
     @XmlIDREF
     private DefaultUnit unit;
 
+    /**
+     * FIXME-1469 Catalog: This is need to be converted to BigDecimal. However, if we do, then TestXmlReader.testCatalogLoad()
+     *   will fails. It is because in some XML (test file), max and or min value is not set, and lead to null value.
```

Comment:
We need to change this field and come up with a fix for the test to pass - perhaps as simple as updating the xml fail with a default value ?